### ISSUE TITLE
problem: Visual traffic light hints are missing from new slider queue UI

### ISSUE THREAD

**Issue by jimkleine** [howlongistheline/howlongistheline.org#61]
The old UI coloured the queue status green/orange/red for a quick visual indication. I'm aware we are making somewhat arbitrary choices re queue depth at this point, but 0 should be green, 50+ should be red and we need to nominate "breakpoints" for green/orange and orange/red boundaries.

solution: Colour the queue description text ("There were nn people in line ...") and or slider control (the latter might be more tricky, since it is an interactive control).

**Comment by gazhayes** [comment]
Yeah even I am annoyed now that the coloured indication is gone!

**Comment by jimkleine** [comment]
I don't have a complete test environment yet (I do have a working local mongodb and Meteor/Mongo plugins installed for VS Code).  As soon as I do, I'll try to pick off some of these presentation issues in quick succession.

**Comment by gazhayes** [comment]
That would be very cool. I've only ever used *nix for development environments, hopefully Meteor works ok on Windows. A quick note for when you start, most of the UI is using onsenui (https://onsen.io/). I've been trying to avoid using any other UI libraries (e.g. bootstrap) because they don't play very nicely. But I have used Material as well without issue.

**Comment by ioioio8888** [comment]
I think I will temporary set the "breakpoints" at 20 for green/orange and 35 for orange/red boundaries first

**Comment by jimkleine** [comment]
@gazhayes This is off-topic, but I'm just responding to your comment re my dev environment.  I'm multi-OS enabled (Windows/MacOS/Ubuntu), but I usually use MacOS for web dev, and I use multiple IDEs.  VS Code (for MacOS) just seemed to have recommended Meteor support when I did a quick search for "best Meteor IDE". The minor code niggles I have been reporting are just a side-effect of me exploring the existing code base to get a feel for style/templating layouts etc.

I followed your environment setup instructions, but Meteor seems to consider itself a bit of a special snowflake and has broken my Homebrew-installed Node and I didn't have time to fix it today.  I'm reluctant to submit pull requests without being able to test, even for trivial HTML stuff.  I've also re-synched my fork with Master, but GitHub still thinks I'm 9 commits behind for some reason: to be investigated.

I've installed the Community edition of [Mongo Management Studio](https://mms.litixsoft.de/index.php?lang=en), to get a quick insight into the current data.  It is complaining that it doesn't support MongoDB v4.x, but is working for me so far.  There is also a MongoDB plugin for VS Code ("Azure Cosmos DB").  I was able to install MongoDB itself with Homebrew without any issues.

Do we have a convenient mechanism for cleaning up the stores data?  Is there a Django equivalent for Meteor: the only ones I found seem to have been abandoned?  Or if I clean up the dev database, is there an efficient way to merge those changes back to the live data?
